runtime/runtime/setup.go

```diff
@@ -146,12 +151,27 @@ func (srv *Server) scrapeMetrics(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
-func (srv *Server) healthz(w http.ResponseWriter, req *http.Request) {
+func (srv *Server) healthz(w http.ResponseWriter, _ *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	w.Write([]byte(`{
-  "message": "Your Encore app is up and running!"
-}
-`))
+
+	bytes, _ := json.Marshal(struct {
+		Code    string `json:"code"`
+		Message string `json:"message"`
+		Details any    `json:"details"`
+	}{
+		Code:    "ok",
+		Message: "Your Encore app is up and running!",
+		Details: struct {
+			AppRevision    string `json:"app-revision"`
+			EncoreCompiler string `json:"encore-compiler"`
+			DeployId       string `json:"deploy-id"`
```

Comment:
JSON case convention is either `snake_case` or `camelCase` and looking through our code base we're primarily using `snake_case`.

```suggestion
			AppRevision    string `json:"app_revision"`
			EncoreCompiler string `json:"encore_rcompiler"`
			DeployId       string `json:"deploy_id"`
```